client/patches/@types+jsurl+1.2.29.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/@types/jsurl/index.d.ts b/node_modules/@types/jsurl/index.d.ts
+index 2b1f0fb..86d4c54 100644
+--- a/node_modules/@types/jsurl/index.d.ts
++++ b/node_modules/@types/jsurl/index.d.ts
+@@ -3,6 +3,8 @@
+ // Definitions by: Alexey Gorshkov <https://github.com/agorshkov23>
+ // Definitions: https://github.com/agorshkov23/DefinitelyTyped
+ 
++export = JSURL;
```

Comment:
@Stephen-ONeil 
`import JSURL from "jsurl"` gives `File @types/jsurl/index.d.ts is not a module`
... I was just playing around and just adding this seem to fix it🤷‍♂️

Reply:
:shrug: 

Opened an issue about it?

Reply:
JSURL repo didn't have a commit since 2016 haha. Probably faster to fix it by using an alternative library